api/spec/v1.yaml

```diff
@@ -1271,7 +1319,52 @@ components:
           type: string
           description: The vote cast.
           example: 'yes'
-    
+
+    RuntimeBlockList:
+      type: object
+      properties:
+        blocks:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeBlock'
+      description: |
+        A list of consensus blocks.
+
+    RuntimeBlock:
+      type: object
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block round.
+          example: *block_height_1 # TODO: add another example, with recent Emerald round
+        hash:
+          type: string
+          description: The block header hash.
+          example: *block_hash_1 # TODO: add another example, with recent Emerald block hash
+        timestamp:
+          type: string
+          format: date-time
+          description: The second-granular consensus time.
+          example: *iso_timestamp_1
+        num_transactions:
+          type: integer
+          format: int32
+          description: The number of transactions in the block.
+          example: 0 # TODO
+        size:
+          type: integer
+          format: int32
+          description: The total byte size of all transactions in the block.
+          example: 0 # TODO
+        gas_used:
+          type: integer
+          format: int64
+          description: The total gas used by all transactions in the block.
```

Comment:
now I wonder, isn't it possible for out-of-transaction stuff to use gas too? why shouldn't we include those

Reply:
I don't know. Good catch! I assumed it was only txs, and I bet Nikhil assumed the same when he wrote this description.
Do you know what other gas consumers there are? I am in favor of listing them (not necessarily exhaustively) here, I expect it will be elucidating to most users.

Reply:
I wrote this description :facepalm:

Reply:
filing #197